CompanyManager.DAL/App.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+        <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
+        <section name="entityFramework"
```

Comment:
Nazwał bym to jakoś bardziej zrozumiale dla ludzi, np. CompanyManagerDBConnectionString , tak żeby było wiadomo co to.